### PR TITLE
pytest: Fix py26 unit tests

### PR DESCRIPTION
**Pull request by tyll**
This needs to be adjusted once tox-lsr and lsr-gh-action-py26 are released.

### CODE REVIEW

File: pytest_extra_requirements.txt [hunk 1]
```diff
@@ -8,4 +8,3 @@ mock ; python_version < "3.0"
 # ansible and dependencies for all supported platforms
 ansible ; python_version > "2.6"
 idna<2.8 ; python_version < "2.7"
-PyYAML<5.1 ; python_version < "2.7"
```
Comment (liangwen12year) [inline pytest_extra_requirements.txt:11]: IMHO, it is better to explain why it is not needed anymore in the commit message.

Comment (tyll) [reply pytest_extra_requirements.txt:11]: yes, you are right. I don't know why it is not needed anymore or why it was added.

Comment (tyll) [reply pytest_extra_requirements.txt:11]: @richm do you know/remember why  PyYAML is there?

Comment (richm) [reply pytest_extra_requirements.txt:11]: > @richm do you know/remember why PyYAML is there?

No, I do not remember

File: .github/workflows/tox.yml [hunk 2]
```diff
@@ -4,7 +4,7 @@ on:  # yamllint disable-line rule:truthy
   - pull_request
   - push
 env:
-  TOX_LSR: "git+https://github.com/linux-system-roles/tox-lsr@2.9.1"
+  TOX_LSR: "git+https://github.com/linux-system-roles/tox-lsr@b7eba23c42e131b988df3dab87e52be51f0ff373"
```
Comment (liangwen12year) [inline .github/workflows/tox.yml:7]: LGTM now.

**Review by liangwen12year — APPROVED** [review]

**Pull request merged**